src/html.js

```diff
@@ -51,6 +51,17 @@ export default class HTML extends React.Component {
             }
             rel='stylesheet'
           />
+          <link
+            href={
```

Comment:
We don't need the `{}` if it is just a string. He curly braces are there to help you do something directly in JavaScript, like string concatenation. We would concat a string if it took the line length over 80 characters.